payroll/models/hr_payslip.py

```diff
@@ -308,6 +308,7 @@ def get_worked_day_lines(self, contracts, date_from, date_to):
         applied for the given contract between date_from and date_to
         """
         res = []
+        date_from, date_to = self._get_param_prev_month(date_from, date_to)
```

Comment:
Please help me understand what's happening here. If `worked_days_from_prev_month` is set then  `date_from` and `date_to` get reset to the previous month's dates? When are the current month's dates evaluated then?

Reply:
The option will create a system parameter `payroll.worked_days_from_prev_month`. If the system parameter value is True, then `date_from, date_to = self._get_param_prev_month(date_from, date_to)` will check replace the dates with the beginning & end of the previous month.